velox/dwio/parquet/tests/ParquetReaderTestBase.h

```diff
@@ -89,11 +89,10 @@ class ParquetReaderTestBase : public testing::Test {
     VectorPtr result = BaseVector::create(outputType, 0, &memoryPool);
 
     while (total < expected->size()) {
+      total += result->size();
```

Comment:
Its confusing that you have incremented total before reading result vector. Would expect it to be after the reader.next(...) call on line 93. Please can you explain the reason.

Reply:
I agree, this is confusing. I have updated the code as you suggested.